parser fix, remove redundant commands

xtpkt opened the pull request:
This resolves a bug involving players specifying "air" and getting dirt blocks. Removing //fill and //fillwall as they are pretty redundant commands in favor of //replace etc.

csmir reviewed (changes requested):
Fill and Fillwall dont actually appear to be removed?

RenderBr commented:
maybe add //fill as an alias for //replace since it's functionality is similar

xtpkt commented:
mfw didnt push worldedit.cs

xtpkt commented:
> maybe add //fill as an alias for //replace since it's functionality is similar

it would literally be replace from air everytime, yeah that could work if you really want to keep them

i.e for `//fill`
`_commandQueue.Add(new Replace(info.X, info.Y, info.X2, info.Y2, e.Player, -1, tiles[0], expression!));`

xtpkt commented:
Came back to it today and the issue was in the expressions' parser not tile ID's lol. 

Removed some of the more useless commands that have been added throughout the plugins lifespan. If you want to really cater with "Alias" commands like these, then please add them with a plugin designed to create them on the fly (such as CommandFactory) rather than clog up a public repo. Their functionality can be found elsewhere, removed:
`//Outline`, `//replace`, `//setgrass`, and `//size`.

..still a horrific mess though.
